D1371R0.pandoc

```diff
@@ -337,7 +337,25 @@ inspect (v) {
 }
 ```
 
-[ _Note:_ Even though `_` is a valid identifier, it does not introduce a name. --- _end note_ ]
+[ _Note:_ Even though `_` is a valid identifier, it does not introduce a name
+as doing so would require all values matched and represented by the wildcard
+to be equal.
+
+It is possible for users to have already introduced `_` as a type or variable
+name in the same scope where an `inspect` statement is used. As a
+highly-visible example, the authors are aware of the use of `_` as a name in
+the popular "Google Mock" library. Idiomatically this is accessed by
+introducing `_` into the current namespace or block scope with a `using`
+directive. Using the wildcard pattern in cases like this is unambiguous
+since the expression pattern requires a `^` introducer for primary expressions. 
+Introduced `_` identifiers will always match against an existing name and 
+un-introduced `_` identifiers will always represent the wildcard pattern. 
+In wildcard cases, an existing `_` name can be used without ambiguity in 
```

Comment:
Remove "In wildcard cases, ". With the current spec, existing `_` can always be accessed from a matched statement.